cmd/sriov-network-config-daemon/start.go

```diff
@@ -276,6 +279,22 @@ func runStartCmd(cmd *cobra.Command, args []string) error {
 	}
 	go nodeWriter.Run(stopCh, refreshCh, syncCh)
 
+	// Init feature gates once to prevent race conditions.
+	defaultConfig := &sriovnetworkv1.SriovOperatorConfig{}
+	err = kClient.Get(context.Background(), types.NamespacedName{Namespace: vars.Namespace, Name: consts.DefaultConfigName}, defaultConfig)
+	if err != nil {
+		if apierrors.IsNotFound(err) {
+			log.Log.Info("default SriovOperatorConfig object not found. waiting for creation.")
```

Comment:
i think nothing is waiting for its creation the function will return and program will exit. is it not the case ?

Reply:
Yeah, you are right. Was a copy&paste fail because the controllers are returning `reconcile.Result{}, nil`, but we have to return an error here. Good catch